Octokit/Models/Request/NewCommitCommentReaction.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class NewCommitCommentReaction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NewCommitCommentReaction"/> class.
+        /// </summary>
+        /// <param name="content">The reaction type.</param>
+        public NewCommitCommentReaction(EnumReaction content)
+        {
+            Ensure.ArgumentNotNull(content, "content");
```

Comment:
Enums can not be null unless specified as <enumType>?
([see here](https://csharp.2000things.com/2011/01/11/208-you-can-make-any-value-type-nullable/) for more information)